src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogView.java

```diff
@@ -160,6 +167,9 @@ private void initDirectorySelection() {
         fileSortCombo.setItems(viewModel.getSorters());
         fileSortCombo.valueProperty().bindBidirectional(viewModel.selectedSortProperty());
         fileSortCombo.getSelectionModel().selectFirst();
+
+
+        directoryPathField.setText(bibDatabase.getMetaData().getDefaultFileDirectory().isPresent() ? bibDatabase.getMetaData().getDefaultFileDirectory().get() : "");
```

Comment:
You can use Optional's orElse method here. https://www.baeldung.com/java-optional-or-else-vs-or-else-get